openbb_platform/core/openbb_core/provider/standard_models/yield_curve.py

```diff
@@ -52,7 +51,23 @@ class YieldCurveData(Data):
         description=DATA_DESCRIPTIONS.get("date", ""),
     )
     maturity: str = Field(description="Maturity length of the security.")
-    rate: float = Field(
-        description="The yield as a normalized percent (0.05 is 5%)",
-        json_schema_extra={"x-unit_measurement": "percent", "x-frontend_multiply": 100},
+
+    @property
+    @computed_field(
+        description="Maturity length as a decimal.",
+        return_type=Optional[float],
     )
+    def maturity_years(self) -> Optional[float]:
+        """Get the maturity in years as a decimal."""
+        if self.maturity is None or "_" not in self.maturity:
+            return None
+
+        parts = self.maturity.split("_")
+        months = 0
+        for i in range(0, len(parts), 2):
+            number = int(parts[i + 1])
+            if parts[i] == "year":
```

Comment:
What do you think about this?

```
parts = self.maturity.split("_")
    months = sum(
        int(parts[i + 1]) * (12 if parts[i] == "year" else 1)
        for i in range(0, len(parts), 2)
    )
    return months / 12
```

Reply:
> What do you think about this?

yup, that's a cleaner way of doing the same thing.